msg/versioned/VehicleCommand.msg

```diff
@@ -14,7 +14,7 @@ uint16 VEHICLE_CMD_NAV_LOITER_TURNS = 18		# Loiter around this MISSION for X tur
 uint16 VEHICLE_CMD_NAV_LOITER_TIME = 19			# Loiter around this MISSION for X seconds |Seconds (decimal)| Empty| Radius around MISSION, in meters. If positive loiter clockwise, else counter-clockwise| Desired yaw angle.| Latitude| Longitude| Altitude|
 uint16 VEHICLE_CMD_NAV_RETURN_TO_LAUNCH = 20		# Return to launch location |Empty| Empty| Empty| Empty| Empty| Empty| Empty|
 uint16 VEHICLE_CMD_NAV_LAND = 21			# Land at location |Empty| Empty| Empty| Desired yaw angle.| Latitude| Longitude| Altitude|
-uint16 VEHICLE_CMD_NAV_TAKEOFF = 22			# Takeoff from ground / hand |Minimum pitch (if airspeed sensor present), desired pitch without sensor| Empty| Empty| Yaw angle (if magnetometer present), ignored without magnetometer| Latitude| Longitude| Altitude|
+uint16 VEHICLE_CMD_NAV_TAKEOFF = 22			# Takeoff from ground / hand |Unused (FW pitch from FW_TKO_PITCH_MIN) |Unused|Unused|Yaw angle in NED [deg] if yaw source available, ignored otherwise|Latitude (WGS-84)|Longitude (WGS-84)|Altitude AMSL [m]|
```

Comment:
```suggestion
uint16 VEHICLE_CMD_NAV_TAKEOFF = 22			# Takeoff from ground / hand |Unused (FW pitch from FW_TKO_PITCH_MIN)|Unused|Unused|Yaw angle in NED [deg] if yaw source available, ignored otherwise|Latitude (WGS-84)|Longitude (WGS-84)|Altitude AMSL [m]|
```